extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerListWorkspacesCommand.java

```diff
@@ -10,37 +10,47 @@
  ******************************************************************************/
 package mil.nga.giat.geowave.cli.geoserver;
 
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
 
 import mil.nga.giat.geowave.core.cli.annotations.GeowaveOperation;
 import mil.nga.giat.geowave.core.cli.api.Command;
 import mil.nga.giat.geowave.core.cli.api.DefaultOperation;
 import mil.nga.giat.geowave.core.cli.api.OperationParams;
-
+import mil.nga.giat.geowave.core.cli.operations.config.options.ConfigOptions;
 import net.sf.json.JSONArray;
 import net.sf.json.JSONObject;
 
+import com.beust.jcommander.JCommander;
 import com.beust.jcommander.Parameters;
 
-@GeowaveOperation(name = "listws", parentOperation = GeoServerSection.class)
+@GeowaveOperation(name = "listws", parentOperation = GeoServerSection.class, restEnabled = GeowaveOperation.RestEnabledType.POST)
 @Parameters(commandDescription = "List GeoServer workspaces")
 public class GeoServerListWorkspacesCommand extends
-		DefaultOperation implements
+		DefaultOperation<List<String>> implements
 		Command
 {
 	private GeoServerRestClient geoserverClient = null;
 
 	@Override
 	public boolean prepare(
 			OperationParams params ) {
-		super.prepare(params);
```

Comment:
`super.prepare(params);` got eliminated, it seems like it needs to be there (at least for the CLI to work the same, presumably the services should be similar too).